tests/fast-integration/kyma-environment-broker/helpers.js

```diff
@@ -0,0 +1,73 @@
+const fs = require("fs");
+const { 
+  wait,
+  debug,
+  getEnvOrThrow,
+} = require("../utils");
+const { 
+  expect 
+} = require("chai");
+
+const SHOOT_KUBECONFIG_PATH = getEnvOrThrow("SHOOT_KUBECONFIG");
+
+async function provisionSKR(keb, gardener, instanceID, name) {
+  const resp = await keb.provisionSKR(name, instanceID);
+  expect(resp).to.have.property("operation");
+
+  const operationID = resp.operation
+  const shootName = resp.dashboard_url.split(".")[1];
+  debug(`Operation ID ${operationID}`, `Shoot name ${shootName}`);
+
+  await ensureOperationSucceeded(keb, instanceID, operationID);
+
+  const shoot = await gardener.getShoot(shootName);
+  debug(`Compass ID ${shoot.compassID}`);
+
+  fs.writeFileSync(SHOOT_KUBECONFIG_PATH, shoot.kubeconfig);
```

Comment:
I would consider not storing shoot kubeconfig in the file. Wouldn't it be cleaner if you just initialize k8s client from the string?

Reply:
done